### DISCUSSION TITLE
Releasing v1.0.0

### DISCUSSION THREAD

**Discussion by tig** [migueldeicaza/NStack#70] [General]
All - esp @migueldeicaza 

We've got a few fixes to NStack that we'd like to deploy to support Terminal.Gui.

I've done more work on the repo to make it more modern:

- Now uses gitflow style branch model (main always represents last published package/develop for current work)
- API docs are now generated from `main` via a GitHub Action

NStack has been very stable for a long time and I think it's time it gets a "v1.0" version #.

Any objections to me making the next release (which otherwise would be `0.18`) be "Release v1.0.0"?

**Comment by BDisp** [comment]
For me I agree at all. It's a very good support for Unicode.
